examples/src/main/scala/example/ExampleApi.scala

```diff
@@ -91,13 +92,13 @@ object ExampleApi {
         Subscriptions(exampleService.deletedEvents)
       )
     ) @@
-      maxFields(300) @@               // query analyzer that limit query fields
-      maxDepth(30) @@                 // query analyzer that limit query depth
-      timeout(3 seconds) @@           // wrapper that fails slow queries
-      printSlowQueries(500 millis) @@ // wrapper that logs slow queries
-      printErrors @@                  // wrapper that logs errors
-      apolloTracing() @@              // wrapper for https://github.com/apollographql/apollo-tracing
-      DeferSupport.defer              // wrapper that enables @defer directive support
+      maxFields(300) @@                         // query analyzer that limit query fields
+      maxDepth(30) @@                           // query analyzer that limit query depth
+      timeout(3 seconds) @@                     // wrapper that fails slow queries
+      printSlowQueries(500 millis) @@           // wrapper that logs slow queries
+      printErrors @@                            // wrapper that logs errors
+      apolloTracing() @@                        // wrapper for https://github.com/apollographql/apollo-tracing
+      IncrementalDelivery.aspect(Feature.Defer) // wrapper that enables @defer directive support
```

Comment:
How about making some nice aliases `IncrementalDelivery.defer` and `IncrementalDelivery.stream`?

Reply:
And maybe `IncrementalDelivery.all`

Reply:
Will do today

Reply:
Done